plans/feat-e2e-live.md

```
@@ -0,0 +1,1010 @@
+    actionTimeout: 60_000,
+    navigationTimeout: 60_000,
+  },
+  projects: [{ name: 'chromium', use: { browserName: 'chromium' } }],
+});
+```
+
+各 spec の冒頭で `test.describe.configure({ mode: 'parallel' })` を入れて、 同一ファイル内のテストも並列で走らせる（spec を増やしたら自動で並列度が上がる）。
+
+### `.gitignore` 追記
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)